ui/src/pages/help-center/help-center-card/HelpCenterCard.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, CardContent, Typography, Card, Stack } from '@mui/material';
+import {
+  Support,
+  ForumOutlined,
+  DatasetOutlined,
+  NorthEast,
+  SaveAlt,
+  MapOutlined,
+} from '@mui/icons-material';
+import { KnowledgeBaseIcon } from 'icons';
+import { FC, ReactNode, useCallback } from 'react';
+import { CARD_IDS, START_ICON } from '../HelpCenter.constants';
+import { HelpCenterCardProps, HelpCardButton } from './HelpCenterCard.types';
+
+export const HelpCenterCard: FC<HelpCenterCardProps> = ({ card }) => {
+  const { id, title, borderColor, description, buttons } = card;
+
+  const getIcon = useCallback((cardId: string): ReactNode => {
+    switch (cardId) {
+      case CARD_IDS.pmmDocs:
+        return <KnowledgeBaseIcon />;
+      case CARD_IDS.support:
+        return <Support />;
+      case CARD_IDS.forum:
+        return <ForumOutlined />;
+      case CARD_IDS.pmmDump:
+        return <DatasetOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const getButtonStartIcon = useCallback((iconName?: string): ReactNode => {
+    switch (iconName) {
+      case START_ICON.download:
+        return <SaveAlt />;
+      case START_ICON.map:
+        return <MapOutlined />;
+      default:
+        return null;
+    }
+  }, []);
+
+  const onButtonClick = useCallback((button: HelpCardButton) => {
+    if (button.target) {
+      window.open(button.url, button.target, 'noopener,noreferrer');
+    } else {
+      // TODO: use react router once the grafana iframe is in place
+      button.url && window.location.assign(button.url);
+    }
+  }, []);
+
+  return (
+    <Card key={id} data-testid={`help-card-${id}`} variant="outlined">
+      <CardContent
+        sx={{
+          px: 2,
+          '&:last-child': {
+            paddingBottom: '16px',
```

Comment:
I think you can use `2` in place of `16px`, it will use the spacing system from MUI.